pyproject.toml

```diff
@@ -62,6 +64,7 @@ test = [
   "pytest-cov>=4.0.0",
   "types-tqdm"
 ]
+toml = ["tomli; python_version < '3.11'"]
```

Comment:
this is not needed for Build-time dependency mode

Reply:
done